README.md

```diff
@@ -33,6 +33,12 @@
   <p>API Announcement!</p>
 </h3>
 
+We are thrilled to announce the launch of our new API with Unstructured's Chipper model to deliver superior performance when processing high-resolution, complex documents. To start using this API request with Chipper model, you can utilize the `hi_res` strategy as stated in the API documentation [here](https://unstructured-io.github.io/unstructured/api.html#strategies).
```

Comment:
Can you update the verbiage here and in the API docs to make it clear that the Chipper release is a Beta and not yet recommended for production use? In the README announcement, you can also invite users to reach out to us on Slack if they're interested in being Beta testers.

cc: @LaverdeS @jackretterer